README.md

```diff
@@ -1,3 +1,166 @@
 # Cluster Node Tuning Operator
 
-The Cluster Node Tuning Operator manages cluster node-level tuning for [OpenShift](https://openshift.io/).
+The Cluster Node Tuning Operator manages cluster node-level tuning for
+[OpenShift](https://openshift.io/).
+
+The majority of high-performance applications require some
+level of kernel tuning.  The operator provides a unified
+management interface to users of node-level sysctls and more
+flexibility to add [custom tuning](#custom-tuning-specification)
+specified by user needs.  The operator manages containerized
+[tuned](https://github.com/redhat-performance/tuned/)
+daemon for [OpenShift](https://openshift.io/) as
+a Kubernetes DaemonSet.  It ensures [custom tuning
+specification](#custom-tuning-specification) is passed to all
+[containerized tuned](https://github.com/openshift/openshift-tuned)
+daemons running in the cluster in the format they understand.
+The daemons run on all nodes in the cluster, one per node.
+
+
+## Deploying the node tuning operator
+
+The operator is deployed by applying the manifests in the operator's
+`/manifests` directory.  It tries to be useful out of the box by
+creating a default deployment and configuration
+([CR](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/))
+for the tuned daemons.  The following shows the default CR created
+by the operator on a cluster with the operator installed.
+
+```
+$ oc get Tuned
+NAME      AGE
+default   1h
+```
+
+## Custom tuning specification
+
+For an example of a tuning specification, please refer to
+`/assets/tuned/07-cr-tuned.yaml` in the operator's directory or to
+the resource created in a live cluster by:
+
+```
+oc get Tuned/default -o yaml
+```
+
+The custom resource for the operator has two major sections.  The first
+section `profile:` is a list of tuned profiles and their names, the second
+`recommend:` defines the profile selection logic.
+
+Multiple custom tuning specifications can co-exist as multiple CRs
+in the operator's namespace.  The existence of new or deletion of
```

Comment:
The existence of new CRs or the deletion of old CRs is detected by the operator. All existing custom tuning specifications are merged and appropriate objects for the containerized tuned daemons are updated.